### DISCUSSION TITLE
二级标题前书写正文，会使二级标题换页的问题

### DISCUSSION THREAD

**Discussion by SiriusFang2020** [sikouhjw/gdutthesis#28] [Q&A] [answered]
如图所示，如果在二级标题前书写了较多的正文，虽然还未铺满整页，但还是会导致二级标题换页
![image](https://user-images.githubusercontent.com/59632817/157400187-d0ae4e36-8383-4c63-a3f4-2a0ac72331cb.png)

**Comment by sikouhjw** [comment, marked as answer]
这不是 bug，这是『排版规则』，也就是『章节标题』不会出现在页尾，它后面必须接正常文本。

我的建议是，如果你或导师看不惯，就想办法加减文字。不改也不会有问题。

PS：为什么 Word 好像不会这样呢？因为大部分人使用 Word 都是在『正文』样式上手动修改。Word 自带样式的『标题 1』就设置了『与下段同页』，因此 Word 也遵守上述规则。

**Comment by SiriusFang2020** [comment]
好的，多谢🙏
